src/main/java/com/stirante/RuneChanger/model/RunePage.java

```diff
@@ -9,9 +9,11 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.stirante.RuneChanger.gui.SettingsController.showWarning;
+
 public class RunePage {
-    private final List<Rune> runes = new ArrayList<>(6);
-    private final List<Modifier> modifiers = new ArrayList<>(3);
+    private List<Rune> runes = new ArrayList<>(6);
+    private List<Modifier> modifiers = new ArrayList<>(3);
```

Comment:
Those lists can still be final. You don't have to create new ArrayList.

Reply:
I'm pretty sure those can't be final, otherwise i'm not sure what you're hinting at.